terraform/examples/daos_cluster/terraform.tfvars.perf.example

```diff
@@ -26,11 +26,34 @@ server_daos_scm_size       = 45
 # server_os_disk_type        = "pd-ssd"
 
 # pools = [
-#  {
-#    pool_name = "test_pool"
-#    pool_size = "1TB"
-#    containers = [ "test_container1", "test_container2" ]
-#  }
+#   {
+#     name = "test_pool",
+#     size = "1TB",
+#     tier_ratio = 3
+#     acls = [
+#       "A::OWNER@:rwdtTaAo",
+#       "A:G:GROUP@:rwtT"
+#     ],
+#     properties = [
+#       "reclaim:disabled"
+#     ],
+#     containers = [
+#       {
+#         name = "test_container1",
+#         type = "POSIX",
+#         acls = [
+#           "A::OWNER@:rwdtTaAo",
+#           "A:G:GROUP@:rwtT"
```

Comment:
The above 2 lines seem to permission for the Owner and Group. 

It would be much better to have the Owner and Group set in this not just their default policy.

Please make a container with both root and your user account and I think you see what I am suggesting.